asf/sam0/include/samc21/component/dac.h

```diff
@@ -133,7 +133,7 @@ typedef union {
 typedef union {
   struct {
     uint8_t  UNDERRUN:1;       /*!< bit:      0  Underrun Interrupt Enable          */
-    uint8_t  EMPTY:1;          /*!< bit:      1  Data Buffer Empty Interrupt Enable */
+    uint8_t  EMPTYDB:1;        /*!< bit:      1  Data Buffer Empty Interrupt Enable */
```

Comment:
The pattern is compress the naming, for instance

SYNCRDY Synchronization Ready Interrupt
so, we need an alternative in the same form to be consistent.

It seems that below is the less worst between what I could check
DBEMPTY

As a remind, you need to rename this on all sam0 series.